src/kakarot/accounts/eoa/library.cairo

```diff
@@ -128,10 +128,14 @@ namespace ExternallyOwnedAccount {
 
         // If destination is 0, we are deploying a contract
         if (destination == 0) {
+            let (tx_info) = get_tx_info();
```

Comment:
`CREATE` opcodes utilize the contract account nonce and not the eoa nonce, so that should not be an issue, right?
I think you might have a point if we do a multicall in an eoa.

I discussed this in DMs with elias and he posted the summary in the issue:
https://github.com/sayajin-labs/kakarot/issues/505

Reply:
indeed, and I even commented this thread ... sorry for the irrelevant comment